test/array.test.js

```diff
@@ -55,7 +58,7 @@ test('Array shift, unshift', () => {
   let a = newArray();
   let before = a.contents;
 
-  a.unshift(a.shift());
+  a.unshift(a.shift(), a.shift());
```

Comment:
Please keep a single-param version of the function call in the test. You can add a multi-param version, but the most simple version of the usage should also be covered in the test.